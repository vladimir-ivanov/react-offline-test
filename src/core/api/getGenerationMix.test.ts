import { getGenerationMix } from './getGenerationMix';

describe('getGenerationMix()', () => {
    it('should return the mocked data', (done) => {
        getGenerationMix().then(data => {
            expect(data).toMatchSnapshot();
            done();
        });
    });
});
