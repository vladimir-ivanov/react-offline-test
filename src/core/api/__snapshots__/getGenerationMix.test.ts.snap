// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getGenerationMix() should return the mocked data 1`] = `
Object {
  "from": "2019-08-12T12:30Z",
  "generationmix": Array [
    Object {
      "fuel": "biomass",
      "perc": 4.8,
    },
    Object {
      "fuel": "coal",
      "perc": 2.5,
    },
    Object {
      "fuel": "imports",
      "perc": 8.7,
    },
    Object {
      "fuel": "gas",
      "perc": 46.5,
    },
    Object {
      "fuel": "nuclear",
      "perc": 16.1,
    },
    Object {
      "fuel": "other",
      "perc": 0.3,
    },
    Object {
      "fuel": "hydro",
      "perc": 0.9,
    },
    Object {
      "fuel": "solar",
      "perc": 14.6,
    },
    Object {
      "fuel": "wind",
      "perc": 5.6,
    },
  ],
  "to": "2019-08-12T13:00Z",
}
`;
